src/main/java/sdk/sample/Creation.java

```diff
@@ -0,0 +1,152 @@
+package sdk.sample;
+
+import com.azure.resourcemanager.netapp.fluent.NetAppManagementClient;
+import com.azure.resourcemanager.netapp.fluent.models.CapacityPoolInner;
+import com.azure.resourcemanager.netapp.fluent.models.NetAppAccountInner;
+import com.azure.resourcemanager.netapp.fluent.models.VolumeInner;
+import sdk.sample.common.CommonSdk;
+import sdk.sample.common.ProjectConfiguration;
+import sdk.sample.common.Utils;
+import sdk.sample.model.ModelCapacityPool;
+import sdk.sample.model.ModelNetAppAccount;
+import sdk.sample.model.ModelVolume;
+
+public class Creation
+{
+    public static void createANFResources(ProjectConfiguration config, NetAppManagementClient anfClient)
+    {
+        /*
+          Creating ANF Accounts
+         */
+        Utils.writeConsoleMessage("Creating Azure NetApp Files Account(s)...");
+        if (!config.getAccounts().isEmpty())
+        {
+            config.getAccounts().forEach(account -> createAccount(anfClient, account));
```

Comment:
You should call the waitForANFResource after the creation.

Reply:
There is no need for it because the create functions are synchronous, they will block until the resource has been created.